go-controller/pkg/ovn/master.go

```diff
@@ -93,19 +92,19 @@ func (oc *Controller) Start(identity string, wg *sync.WaitGroup, ctx context.Con
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
 				klog.Infof("Won leader election; in active mode")
-				// run the cluster controller to init the master
 				start := time.Now()
 				defer func() {
 					end := time.Since(start)
 					metrics.MetricMasterReadyDuration.Set(end.Seconds())
 				}()
 
-				if err := oc.StartClusterMaster(); err != nil {
+				if err := cm.Init(); err != nil {
 					klog.Error(err)
 					cancel()
 					return
 				}
-				if err := oc.Run(ctx, wg); err != nil {
+
+				if err := cm.defaultController.Init(ctx); err != nil {
```

Comment:
Here we should have a `cm.Run`, `cm.defaultController.Init(ctx)` should be renamed to `cm.defaultController.Run(ctx)` and be called from within `cm.Run`